src/Components/test/testassets/Components.WasmMinimal/Properties/launchSettings.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Is this change needed?

Reply:
I was inclined to agree with this, and I went as far as to checkout the jennyf19:jennyf/idWebTemplatesUpdate branch and revert 331d122dfefe094334f1f17cb141d9e3862eee9b before merging, only to notice that my VS had automatically created a different launchSettings.json file. I don't think it makes any sense for a Microsoft.NET.Sdk.BlazorWebAssembly project, but it's just going to keep cluttering git's untracked files until we check it in or ignore it.

I'm just going to check it in for now since it seems harmless and I don't want to rekick the build, but we should figure out why VS is generating this. @SteveSandersonMS @dotnet/razor-tooling

Reply:
VS automatically creating a launchsettings file would be a question for @BillHiebert or @tmeschter I believe. I'm not sure why it necessarily doesn't make sense to have one for a Blazor WASM project though, the project system needs to know what to tell the debugger when people press F5, AFAIK. Maybe things can be changed to not write a default to disk though?